src/serviceWorker.js

```diff
@@ -20,40 +20,6 @@ const isLocalhost = Boolean(
     )
```

Comment:
[note] The changes in this file _should_ just be moving the position of the functions around, there shouldn't be any changes to the code within them.